lib/service/Service.ts

```diff
@@ -86,6 +91,25 @@ class Service extends EventEmitter {
     this.version = components.version;
   }
 
+  /** Adds a supported currency. */
+  public addCurrency = async (args: { currency: string, swapClient: SwapClients | number, tokenAddress?: string, decimalPlaces?: number }) => {
+    argChecks.VALID_CURRENCY(args);
+    argChecks.VALID_SWAP_CLIENT(args);
+    const { currency, swapClient, tokenAddress, decimalPlaces } = args;
+
+    await this.orderBook.addCurrency({
+      tokenAddress,
+      swapClient,
+      decimalPlaces: decimalPlaces || 8,
```

Comment:
I think this default value should be in the orderbook instead of the service class.